openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -151,6 +151,10 @@ def update_account_settings(requesting_user, update, username=None):
         changing_full_name = True
         old_name = existing_user_profile.name
 
+    changing_secondary_email = False
```

Comment:
nit: `changing_secondary_email` -> `change_secondary_email`

Reply:
I think it would be better to stick to the existing convention. A similar form is used in other places of the file like `changing_email`, `changing_full_name` etc.